assets/sass/modules/_forms.scss

```diff
@@ -189,6 +189,40 @@
       right: 10px;
       opacity: 0.5;
     }
+
+    .password-toggle {
+
+      .button-off {
+        display: none;
+      }
+
+      .button {
+        right: 5px;
```

Comment:
Interesting - maybe that's why we have a problem in the IE's. I'll investigate it later (I have a couple jiras related to embedded browser + the widget)